RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1429,6 +1429,23 @@ boolean auto_knapsackAutoConsume(string type, boolean simulate)
 	return true;
 }
 
+int auto_spleenFamiliarAdvItemsPossessed() 
+{
+	//returns how many size 4 items from spleen familiars in possession
```

Comment:
nice. could maybe combine it with the code in `auto_familiar.ash` lines 495 to 510
starting with `int available_spleen_items_size = 0;`

Reply:
combined it, same items and your price check seems better than the one it was using